proto/tripcommon.proto

```diff
@@ -175,6 +185,12 @@ message TaggedValue {
     // see https://github.com/valhalla/valhalla/issues/3262
     kTunnel = 49; // static_cast<uint8_t>('1')
     kBridge = 50; // static_cast<uint8_t>('2')
+    kBssName = 51;
+    kBssRef = 52;
+    kBssNetwork = 53;
+    kBssCapacity = 54;
+    kBssSource = 55;
+    kBssOperator = 56;
```

Comment:
what if we combined these into a single record? one interesting idea would be to construct the protobuf object above `BikeShareStationInfo` serialize it to bytes and then b64 encode the bytes and store that as the value. this would make our lives really easy when getting the info back out of the tiles.

another question i have is why dont these fields match the protobuf above?

Reply:
also this s hould use a value of `2`. tunnel and bridge were mistakenly using chars when first implemented